src/views/index.js

```diff
@@ -1,5 +1,11 @@
+// IIFE - immediately invoked function expression
 window.onload = function () {
     const appEl = document.querySelector('#app');
-    appEl.innerHTML = LoginPage();
-    AddEventsLogin();
-}
\ No newline at end of file
+    const isLogged = !!sessionStorage.userLogged;
```

Comment:
Minor. If you're converting to a boolean using `!!` here, but checking against the opposite value just below, on line 5, is there any way to simplify and avoid the extra processing?